R/zchunk_L226.en_distribution_USA.R

```diff
@@ -1,20 +1,35 @@
 #' module_gcam.usa_L226.en_distribution_USA
 #'
-#' Briefly describe what this chunk does.
+#' This chunk creates L226.DeleteSupplysector_USAelec, L226.StubTechCoef_electd_USA, L226.TechShrwt_electd_USA, L226.TechCost_electd_USA,
```

Comment:
Can we make this a little more descriptive and summarize information from below?